app/assets/javascripts/components/composition-notes.jsx

```diff
@@ -0,0 +1,82 @@
+import PropTypes from 'prop-types'
+import Textarea from 'react-textarea-autosize'
+
+class CompositionNotes extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = { notes: props.notes }
+  }
+
+  componentWillReceiveProps(nextProps) {
+    this.setState({ notes: nextProps.notes })
+  }
+
+  onCompositionNotesChange(event) {
+    this.setState({ notes: event.target.value })
+  }
+
+  saveNotes(event) {
+    if (this.state.isRequestOut) {
+      return
+    }
+    if (event) {
+      event.target.blur()
+    }
```

Comment:
@cheshire137 I'm not understanding why this `blur` is required and I'm curious :) Mind explaining this to me?

What I understand now is the only time we have an event passed to `this.saveNotes` is when the `save-notes-button` is clicked. The target of the event is the button, and so I'm not sure what `blur` adds in that case?